content/general/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Data Types'
+Description: 'Data is classified into data types that tell the compiler how the data is intended to be used.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/build-web-apps-with-asp-net'
+---
+
+Data is foundational in programming and is classified into different types to indicate how it can be interpreted and used in the compiler.
+
+Data types can be simple or complex. Simple data types are the most basic types and usually hold a single value. A complex data type can be composed of multiple data types and often group multiple values.
```

Comment:
@ethan-master-coding Concerning this line and the tables below, I think this "simple/complex" difference can be unpacked a bit further. For Docs entries, let's try to avoid using terms like "basic" and "simple". Data types can be categorized in three distinct ways:

- Primitive data types that contain a single value such as bytes, booleans, character, and numbers.
- Composite data types that made of multiple primitive types, such as arrays, strings of characters, and objects.
- Other unique types such as pointers and functions.

Reply:
Got it. I will keep that in mind when making future entries/edits.

Reply:
Appreciated! After we  we should be good to go:

- Line 17: Remove "simple" and "basic" language and replace with sentence or two about how data types are generally allocated into three groups: primitive types, composite types, and other unique types.

- Lines 19-37: Rename the two headers for these two tables (as well as add one more header + table), here is what I'm suggesting:

```
## Primitive Types

[Table with primitive types - Number, Character, Boolean]

## Composite Types

[Table with composite types - Array, String, Object]

## Unique Types

[Table with unique types - Null, Undefined, Functions]
```

Reply:
@Dusch4593 Missed this until now. I'm working on this.

Reply:
@Dusch4593 Finished the changes. Take a look now.